src/sentry/workflow_engine/models/json_config.py

```diff
@@ -0,0 +1,22 @@
+from abc import abstractproperty
+from typing import Any
+
+from django.db import models
+from jsonschema import ValidationError, validate
+
+
+class JSONConfigBase(models.Model):
+    config = models.JSONField(default=dict, blank=True, null=True)
```

Comment:
You can define this as `models.JSONField(db_default=dict)`. No need for it to be nullable